### PR TITLE
test-runner: self-contained dockerfile

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

It doesn't depend on k8s.io/test-infra anymore. This is an "almost"
naive import of the `kubeins-e2e` and `bootstrap` images from
k8s.io/test-infra.

Will need to be cleaned up as we go.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @afrittoli @bobcatfish 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by afrittoli** [comment]
goomba at your service! </p>Cloud not build the requested image. Please check the <a href="http://35.222.249.224/?buildid=783b86be-7e34-48e5-ae41-217a60436923&namespace=mario">build logs</a>

**Comment by afrittoli** [comment]
goomba at your service! </p>Cloud not build the requested image. Please check the <a href="http://35.222.249.224/?buildid=7d5e4b22-944f-4cc6-a632-0d7204f4fe31&namespace=mario">build logs</a>

**Comment by afrittoli** [comment]
tekton at your service! </p>Here is the image you requested: <a href="http://gcr.io/tekton-releases/dogfooding/test-runner:mario">built image</a>|<a href="http://35.222.249.224/?buildid=08f34481-d2bd-4075-93d4-49a711f6c63b&namespace=mario">build logs</a>

File: tekton/images/test-runner/Dockerfile [hunk 1]
```diff
@@ -12,9 +12,162 @@
...
+# common util tools
+# https://github.com/GoogleCloudPlatform/gsutil/issues/446 for python-openssl
+RUN apt update && apt install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    file \
+    git \
+    jq \
+    mercurial \
```
Comment (chmouel) [inline tekton/images/test-runner/Dockerfile:34]: ???

**Comment by chmouel** [comment]
/lgtm

i have no idea what this PR does but if the end goal is to can move away from python2 then it's a huge +1 from me :D

File: tekton/images/test-runner/Dockerfile (hunk 1 above)
Comment (vdemeester) [reply tekton/images/test-runner/Dockerfile:34]: `kubeins-e2e` and `bootstrap` from k8s.io/test-infra :sweat_smile: 
I'll clean some stuff later on :wink:

**Comment by dibyom** [comment]
Looks like some merge conflicts with python3? Otherwise lgtm!
/approve

**Pull request merged**